sample/app/google-services.json

```diff
@@ -0,0 +1,42 @@
+{
+  "project_info": {
+    "project_number": "433406026043",
+    "firebase_url": "https://mparticle-5fe9f.firebaseio.com",
+    "project_id": "mparticle-5fe9f",
+    "storage_bucket": "mparticle-5fe9f.appspot.com"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:433406026043:android:776f09f1f45aa15b",
+        "android_client_info": {
+          "package_name": "com.rokolabs.mparticle.sample"
+        }
+      },
+      "oauth_client": [
+        {
+          "client_id": "433406026043-tm1lsasjm86cenkridltnjoh07bo65g4.apps.googleusercontent.com",
+          "client_type": 3
+        }
+      ],
+      "api_key": [
+        {
+          "current_key": "AIzaSyCj3pNSgTm7j42DrtdpYkwrPF2hVMfDNtM"
```

Comment:
are you OK with having these keys public?

Reply:
this was just for the sample app - we'll update PR to remove the google-services.json file entirely since the end-customer would need to insert their *own* google-services.json file anyway.